go/genkit/genkit.go

```diff
@@ -23,6 +23,35 @@ import (
 	"github.com/firebase/genkit/go/core"
 )
 
+// Options are options to [Init].
+type Options struct {
+	DevAddr string
```

Comment:
Dev server address/port probably should not be allowed to be specified programmatically. They are managed by the CLI and passed in via env var.

Reply:
Done.